homeassistant/config.py

```diff
@@ -366,6 +370,12 @@ def set_time_zone(time_zone_str):
     if CONF_TIME_ZONE in config:
         set_time_zone(config.get(CONF_TIME_ZONE))
 
+    # init whitelist external dir
+    hac.whitelist_external_dirs.add(hass.config.path('www'))
```

Comment:
We actually need to set it to a new set here, because this method is also called when we reload the core config.